[2025-02-09T19:52:13.485+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-09T19:52:13.498+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: branch_dim_gen.generate_branch_dim_data manual__2025-02-09T19:52:11.381168+00:00 [queued]>
[2025-02-09T19:52:13.506+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: branch_dim_gen.generate_branch_dim_data manual__2025-02-09T19:52:11.381168+00:00 [queued]>
[2025-02-09T19:52:13.507+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-09T19:52:13.518+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_branch_dim_data> on 2025-02-09 19:52:11.381168+00:00
[2025-02-09T19:52:13.526+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=285) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-09T19:52:13.528+0000] {standard_task_runner.py:72} INFO - Started process 291 to run task
[2025-02-09T19:52:13.526+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'branch_dim_gen', 'generate_branch_dim_data', 'manual__2025-02-09T19:52:11.381168+00:00', '--job-id', '946', '--raw', '--subdir', 'DAGS_FOLDER/brnach_dim_gen.py', '--cfg-path', '/tmp/tmpx47qt7kf']
[2025-02-09T19:52:13.528+0000] {standard_task_runner.py:105} INFO - Job 946: Subtask generate_branch_dim_data
[2025-02-09T19:52:13.566+0000] {task_command.py:467} INFO - Running <TaskInstance: branch_dim_gen.generate_branch_dim_data manual__2025-02-09T19:52:11.381168+00:00 [running]> on host a541c7aa2e43
[2025-02-09T19:52:13.631+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='branch_dim_gen' AIRFLOW_CTX_TASK_ID='generate_branch_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-09T19:52:11.381168+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-09T19:52:11.381168+00:00'
[2025-02-09T19:52:13.632+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-09T19:52:13.653+0000] {logging_mixin.py:190} INFO - CSV file './branch_dim_large_data.csv' with 50 rows has been generated successfully.
[2025-02-09T19:52:13.653+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-09T19:52:13.659+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-09T19:52:13.659+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=branch_dim_gen, task_id=generate_branch_dim_data, run_id=manual__2025-02-09T19:52:11.381168+00:00, execution_date=20250209T195211, start_date=20250209T195213, end_date=20250209T195213
[2025-02-09T19:52:13.703+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-09T19:52:13.716+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-09T19:52:13.718+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
